addons/survey/views/survey_templates.xml

```diff
@@ -15,108 +15,58 @@
         </xpath>
     </template>
 
-    <!-- First page of a survey -->
-    <template id="survey_page_start" name="Survey: start page">
+    <template id="survey_page_fill" name="Survey: main page (take survey)">
         <t t-call="survey.layout">
             <div class="wrap">
-                <div class="oe_structure" id="oe_structure_survey_init_1"/>
-                <div class="container">
+                <div class="oe_structure" id="oe_structure_survey_survey_1"/>
+                <div class="container o_survey_form">
                     <t t-if="answer.test_entry" t-call="survey.survey_button_form_view" />
-                    <div class='jumbotron mt32'>
-                        <h1 t-field='survey.title' />
-                        <div t-field='survey.description' class="oe_no_empty"/>
-                        <div t-if="survey.is_time_limited">
-                            <p>
-                                <span>Time limit for this survey: </span>
-                                <span class="font-weight-bold text-danger" t-field="survey.time_limit" t-options="{'widget': 'duration', 'unit': 'minute'}"></span>
-                            </p>
-                        </div>
-                        <a role="button" class="btn btn-primary btn-lg" t-att-href="'/survey/fill/%s/%s' % (survey.access_token, answer.access_token)">
-                            <t t-if="survey.certification">
-                                Start Certification
-                            </t>
-                            <t t-else="">
-                                Start Survey
-                            </t>
-                        </a>
-                    </div>
+
+                    <t t-call="survey.survey_fill_header" />
+                    <t t-call="survey.survey_fill_form" />
                 </div>
-                <div class="oe_structure" id="oe_structure_survey_init_2"/>
+                <div class="oe_structure" id="oe_structure_survey_survey_2"/>
```

Comment:
bioutiful name btw :smile: 
(_2 ?)

Reply:
Existed before me.
Just moved it :(

Reply:
I know, but maybe we could take to opportunity #tankaffaire to rename thoses (_1 and _2 to be more self speaking about usage of those..)

Reply:
Well if I had any idea what they're used for I would gladly rename them :D